src/main/java/gregtech/api/unification/material/MaterialIconType.java

```diff
@@ -85,6 +85,7 @@
     frameGt,
     frameSide,
     frameTop,
+    metalCasing,
```

Comment:
i'd advice using gtMetalCasing here, since such ore prefix can be used in other mods and will mean different thing.

Reply:
ok I change that